Assets/LeapMotionModules/UIInput/Scripts/CompressibleUI.cs

```diff
@@ -48,8 +48,12 @@ public struct Layer {
     public float PushPaddingDistance = 0.01f;
     //public bool RetractWhenOutsideofTouchingDistance = false;
 
-    [Tooltip("Triggered when the layers that have 'TriggerLayerEvent' enabled are beginning to be collapsed")]
-    public UnityEvent LayerCollapseStateChange;
+    [Tooltip("Triggered when the layers that have 'TriggerLayerEvent' enabled go from 'Expanded' to 'Partially Collapsed'")]
```

Comment:
Should either use "Partially Collapsed" below or "Partially Expanded" here instead